### ISSUE TITLE
Rotation problem

### ISSUE THREAD

**Issue by gimmel** [jomjol/AI-on-the-edge-device#847]
Something is going wrong with rotation, I perfectly placed the ROIs for analog and digital areas, but on the main page, the viewboxes are unaligned.
![ROI](https://user-images.githubusercontent.com/135626/177112766-b55318bf-11d2-4b14-aa8c-2950bc54e570.jpeg)
![Main](https://user-images.githubusercontent.com/135626/177112772-8803f79d-3cb6-4323-bab1-92cb11d0ac6c.jpeg)

**Comment by friedpa** [comment]
The upper right alignment point is not good. Just take a black sticker (round) and put it instead of the current alignment point

**Comment by pos-ei-don** [comment]
I don't think that it has to do with aligment-points.
I have a black point, and do see this aswell!
But... it seems that it work anyway, so i thought that this might be a visible-problem only (some times).

**Comment by gimmel** [comment]
No, it’s not a display bug. The numbers are cropped exactly as shown in the overview, not as defined in the ROI Screens. That’s the reason why the 7 is not recognized and the 5 is recognized as 3.
![Wrong crop](https://user-images.githubusercontent.com/135626/177181607-ddd16451-9891-42d7-8628-918990f998b7.jpg)

This problem might be caused by [Fine Alignment resets to 0](https://github.com/jomjol/AI-on-the-edge-device/issues/846)

**Comment by jomjol** [comment]
Hi,
this is clearly a problem of not found references. In this case the prealignment fails with arbitrary coordinates for the rotation. If you use the "8" and the "m³" I'm not surprised about this, because the contrast is rather low and the algo is working as a single autocorrelation with shifting the image.

The prealignment angle has nothing to do with this. It's only purpouse is to make a "perfect" reference image with a rotation in 0.2° steps. It is not used for anything else and therefore not stored. Therefore it is alway "0", when you restart the image alignment.

If you follow the suggestion of @friedpa you should improve the situation.

**Comment by gimmel** [comment]
Thanks jomjol.
Can you explain in more detail what the alignment points do exactly? I was sure you rotate the image correctly with the pre-rotation and the alignment points are there to re-adjust for slight shifts, similar to a 3D tracker. I would have thought the dots wouldn't be necessary at all if the image area didn't change.

So, I'm going to go down to the basement now with a sharpie and try to open up my construction of pizza box and duct tape. ;-)

**Comment by gimmel** [comment]
I put two matte black markers on the water meter, realigned the camera to 0 degrees, and restarted the setup.
However, the image ends up being randomly rotated again.

![0 degree](https://user-images.githubusercontent.com/135626/177205153-86353b1f-a7e9-407f-9964-375e6e8b7497.jpg)

Then I did a reboot. And suddenly everything worked as expected.
I didn't realize that you had to reboot after making adjustments. Sorry for the unnecessary thread.
